app/controllers/items_controller.rb

```diff
@@ -23,6 +23,28 @@ def new
   def edit
   end
 
+  # GET /items/borrowed
+  # lists all items borrowed by the current user
+  def borrowed_items
+    @items = current_user.lendings.where(completed_at: nil).map(&:item)
+    render :borrowed_items
```

Comment:
- redundancy: Rails should automatically render the `borrowed_items.html.erb` upon GET request to this route.